### DISCUSSION TITLE
librehardwaremonitor VS openhardwaremonitor

### DISCUSSION THREAD

**Discussion by NolonQ** [LibreHardwareMonitor/LibreHardwareMonitor#2059] [General]
What EXACTLY is the difference between openhardwaremonitor and librehardwaremonitor?
Judging from the GUI it looks 1:1 the same, does it not?

**Comment by jeffdamp-wave** [comment]
Looks like openhardwaremonitor is both the monitor and hardware interface in one .exe. and hasn't been updated in 4 years. librehardwaremonitor splits out the library and .exe. Other programs such as FanControl use this repos library.

**Comment by NolonQ** [comment]
> Looks like openhardwaremonitor is both the monitor and hardware interface in one .exe. and hasn't been updated in 4 years. librehardwaremonitor splits out the library and .exe. Other programs such as FanControl use this repos library.

Thank you, so OHM is simply outdated then i'll stick to LHW, thanks again.
